tests/e2e/screenObjects/cart.js

```diff
@@ -52,14 +53,12 @@ class CartContent extends Base{
   }
 
   /**
-   * Remove an item from the cart
-   *
-   * @param {number|string} needle
+   * Remove the first item from the cart
    *
    * @return {void}
    */
-  removeSwagItem(needle){
-    return this.swagItem(needle).$(`~test-${SELECTORS.cartContent.cartItem.remove}`).click();
+  removeSwagItem(){
+    return this.swagItems[0].$(`~test-${SELECTORS.cartContent.cartItem.remove}`).click();
```

Comment:
I'm curious why this choice, to always remove the first item and not give the caller a choice any more - it simplifies the code (although there is still an ArrayIndexOutOfBoundsException possible if swagItems is empty since we don't check length before indexing), but does it remove flexibility in testing?

Reply:
Good question, the new layout now only shows 1 full item and might give 2, the second will only hold a wrapper, and not the real content, that's why I default to the first. This might change in the future because I've requested a layout change. For now we need to have this to get the automation working